x-pack/plugins/cloud_security_posture/server/saved_objects/csp_settings.ts

```diff
@@ -16,8 +16,15 @@ export const cspSettings: SavedObjectsType = {
   indexPattern: SECURITY_SOLUTION_SAVED_OBJECT_INDEX,
   hidden: true,
   namespaceType: 'agnostic',
-  schemas: {
-    '8.12.0': cspSettingsSchema,
```

Comment:
With this PR, tests were catching the error. Fixed by moving the schema to a `modelVersion`.

Reply:
cc @CohenIdo FYI

Reply:
for more context, here are the docs on the new `modelVersions` https://github.com/elastic/kibana/blob/cadabe5be9f483c6c01ffd4da49ca6bba101b6b1/packages/core/saved-objects/core-saved-objects-server/docs/model_versions.md

The change makes sense according to the docs, will test a bit manually as well

Reply:
FYI, I've already [updated the public docs](https://github.com/elastic/kibana/pull/176970) to include most of what's in the `model_versions.md` one, aimed at 8.14.0.